### PR TITLE
Add basic KeyMap customization support

### PR DESCRIPTION
**Pull request by plrenaudin**
This PR is adding a basic KeyMap customization support.
Changed the "Privacy Settings" to "Settings" and added the KeyMap Settings section
The key combinations are stored with the user settings.

This is a great project, thank you @PalmerAL !

### CODE REVIEW

**Comment by PalmerAL** [comment]
This looks nice!

settings.js isn't really meant to hold default settings - the ```list``` item is used as a cache so the database doesn't have to be queried. This causes a race condition where the keymap will sometimes fail to load, since the value in the database will be used if the settings haven't loaded yet.

I think a better solution would be to create a file in ```/util/defaultKeyMap.js```, which does something like this: 

```
var defaultKeyMap = {
			"addPrivateTab": "shift+mod+p",
```

Then, in ```keybindings.js```, the code could look like this:

```
settings.get('keyMap', function (keyMap) {
  if(!keyMap) {
    keyMap = defaultKeyMap;
  }
```

**Comment by PalmerAL** [comment]
The heading in the settings page should probably have the same style as the "content blocking" heading. Also, I think some changes to the spacing would be good, as well as adding a subtitle underneath the heading:

screen shot 2016-05-14

File: pages/settings/settings.js [hunk 1]
```diff
@@ -6,6 +6,49 @@ var contentTypes = {
...
+	Object.keys(keyMap).forEach(createKeyMapInput);
+});
+
+function createKeyMapInput(action) {
+	var keyMapList = document.getElementById('key-map-list');
+	var key = keyMap[action];
+	var li = document.createElement('li');
+	var label = document.createElement('label');
+	var input = document.createElement('input');
+	label.innerText =	label.htmlFor = action;
```
Comment (PalmerAL) [inline pages/settings/settings.js]: Would it be possible to have the formatted name of the action ("Open Private Tab" instead of "openPrivateTab") here?

**Comment by plrenaudin** [comment]
Thanks, it makes more sense this way.

**Comment by plrenaudin** [comment]
@PalmerAL Let me know if you want something else for this to be merged.

**Comment by PalmerAL** [comment]
Looks good to me!

Can you please squash this into one commit? After that, I think this is ready to be merged.

**Comment by plrenaudin** [comment]
Done!

**Comment by PalmerAL** [comment]
Thanks!

**Pull request merged**